views/admin.functions.subscribe.options.php

```diff
@@ -12,8 +12,22 @@ function welcome_template_callback( $args ) {
 		echo '<div class="postbox">';
 		echo '<div class="inside">';
 
-		echo '<h3>Welcome Template</h3>';
-		echo '<p>Choose a template to send after a user signs up using the Sailthru Subscribe Widget or shortcode.';
+		echo '<h3>Welcome Email</h3>';
+		echo '
+<p>
+	<strong>Optional. </strong> Choose a template to send after a user signs up using the Sailthru Subscribe Widget. 
+	The email will only be sent when a user does not belong to at least one of the lists selected in the subscribe widget.
+</p>
+<p>
+	If using double opt-in, customers can be added to the appropriate list via 
+	<code>
+		<a href="https://getstarted.sailthru.com/developers/zephyr-functions-library/signup_confirm/">signup_confirm</a>
+	</code>. 
```

Comment:
This additional spacing makes the code portion of the text visually look weird onsite. Can you change to this:
```suggestion
		<code><a href="https://getstarted.sailthru.com/developers/zephyr-functions-library/signup_confirm/">signup_confirm</a></code>.
```